src/shutdown/shutdown.c

```diff
@@ -400,6 +400,9 @@ int main(int argc, char *argv[]) {
         /* Lock us into memory */
         (void) mlockall(MCL_CURRENT|MCL_FUTURE);
 
+        if (mount(NULL, "/", NULL, MS_REC|MS_PRIVATE, NULL) < 0)
```

Comment:
please add a brief comment here as to why. i.e just say "we want to be able to potentially use MS_MOVE later, and that only works on MS_PRIVATE mounts" or so.